mods/d2k/rules/structures.yaml

```diff
@@ -1038,13 +1040,12 @@ palace:
 	RevealsShroud:
 		Range: 4c768
 	RenderSprites:
-		Image: palace.harkonnen
+		Image: palace.ordos
 		FactionImages:
 			atreides: palace.atreides
-			ordos: palace.ordos
+			fremen: palace.atreides
+			harkonnen: palace.harkonnen
 			corrino: palace.corrino
```

Comment:
The `Corrino palace` exit point is a bit off, could this be updated in this PR? IMO it isn't urgent.

Reply:
Corrino palace isn't meant to be able to build stuff tbh. Considering #14460 maybe we should seperate the palaces.

Reply:
Sounds good, I left a line about it at the issue.